website/blog/bazelcon-2023-recap.mdx

```diff
@@ -0,0 +1,62 @@
+---
+slug: bazelcon-2023
+title: Bazelcon 2023 Recap
+author: Iain Macdonald
+author_title: Software Engineer @ BuildBuddy
+date: 2023-10-30 12:00:00
```

Comment:
wanna make this date Nov 1st, and we'll make noise about it tomorrow?

you don't have to wait to submit it though

Reply:
SG, submitted. For the folks who didn't approve, feel free to send PRs if you see anything you'd like to update!